c7n/resources/swf.py

```diff
@@ -0,0 +1,65 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.utils import local_session
+from c7n import tags
+
+
+@resources.register('swf')
+class SimpleWorkflowDomain(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = arn_type = 'swf'
+        enum_spec = ('list_domains', 'domainInfos', {'registrationStatus': 'REGISTERED'})
+        id = name = 'name'
```

Comment:
needs an arn = "arn" as that appears to be an intrinsic attribute of the domain resource.